### ISSUE TITLE
PlantUML sources is not rendered. 

### ISSUE THREAD

**Issue by kfiry77** [yzane/vscode-markdown-pdf#214]
I successfully generate PDF from markdown. It includes Plantuml. However, inside the PDF, The plant UML is ignored, and it shows only it's PlantUML syntax.  there is only the source code.  I have tried to use in the markdown a link to the server it it works. 
I checked all settings and couldn't find anything about it.

My setup:
Windows 10
markdown-pdf: 1.4.4
VS Code: 1.50.1

**Comment by kfiry77** [comment]
Issue #204 reports the same symptom but with Mermaid. I have tried that and surprisingly  the rendering works great for Mermaid.

**Comment by kfiry77** [comment]
apparently I was using it wrong, since I have used the same notation as PlantUML in markdown requires. : 
` 
```plantuml
@startuml
Alice -> Bob: 
Bob --> Alice:
@enduml
```
`
However, the plantuml code enforces it to ignore it, and treat that fragment as a code, so In order to overcome it I have changed, 
So I have removed the ``` and then it works as expected.  -> I think that this parser should be aware of that since the VS code preview works exactly with ```plantuml notation.  

At the end I was able to align both preview and pdf rendering by setting the configurations as follows: 
`
    "markdown-pdf.plantumlOpenMarker":"```plantuml",
    "markdown-pdf.plantumlCloseMarker":"```",
`

Channing the default as I demonstrate can improve that also.
